proxy/TransformInternal.h

```diff
@@ -24,9 +24,9 @@
 #pragma once
 
 #include "HttpSM.h"
-#include "MIME.h"
-#include "Transform.h"
-#include "P_EventSystem.h"
+//#include "MIME.h"
```

Comment:
Any reason not to just delete these lines?

Reply:
Unintentional. Fixed in next push.